src/ensembl/src/content/app/species-selector/components/species-search-field/SpeciesSearchField.tsx

```diff
@@ -24,6 +24,9 @@ import {
   SearchMatch,
   SearchMatches
 } from 'src/content/app/species-selector/types/species-search';
+
+import GoogleAnalyticsTracking from 'src/services/analytics-service';
```

Comment:
This also should be changed: `GoogleAnalyticsTracking` -> `analyticsService`

Reply:
That's done.